bbot/modules/wpscan.py

```diff
@@ -0,0 +1,263 @@
+import json
+from bbot.modules.base import BaseModule
+
+
+class wpscan(BaseModule):
+    watched_events = ["HTTP_RESPONSE"]
+    produced_events = ["URL_UNVERIFIED", "FINDING", "VULNERABILITY", "TECHNOLOGY"]
+    flags = ["active", "aggressive"]
+    meta = {
+        "description": "Wordpress security scanner. Highly recommended to use an API key for better results.",
+        "created_date": "2024-05-29",
+        "author": "@domwhewell-sage",
+    }
+
+    options = {
+        "api_key": "",
+        "enumerate": "vp,vt,tt,cb,dbe,u,m",
+        "threads": 5,
+        "request_timeout": 60,
+        "connection_timeout": 30,
+        "disable_tls_checks": True,
+        "force": False,
+    }
+    options_desc = {
+        "api_key": "WPScan API Key",
+        "enumerate": "Enumeration Process see wpscan help documentation (default: vp,vt,tt,cb,dbe,u,m)",
+        "threads": "How many wpscan threads to spawn (default is 5)",
+        "request_timeout": "The request timeout in seconds (default 60)",
+        "connection_timeout": "The connection timeout in seconds (default 30)",
+        "disable_tls_checks": "Disables the SSL/TLS certificate verification (Default True)",
+        "force": "Do not check if the target is running WordPress or returns a 403",
+    }
+    deps_apt = ["ruby-rubygems"]
```

Comment:
This package is named differently depending on the distro:

- Ubuntu: `ruby-rubygems`
- Debian: `ruby-rubygems`
- Arch: `rubygems`
- Fedora: `rubygems`

Let's bake that into the ansible playbook like so:

```python
    {
        "name": "Install Rubygems (Non-Debian)",
        "package": {
            "name": "rubygems",
            "state": "present"
        },
        "become": True,
        "when": "ansible_facts['os_family'] != 'Debian'",
    },
    {
        "name": "Install Rubygems (Debian)",
        "package": {
            "name": "ruby-rubygems",
            "state": "present",
        },
        "become": True,
        "when": "ansible_facts['os_family'] == 'Debian'",
    },
```